collections/forest-carbon-monitoring/sandbox-data.md

```diff
@@ -46,9 +46,47 @@ This sandbox collection of <a href="../forest-carbon-monitoring/">Forest Carbon
 
 To purchase data over your own areas and times of interest, <a href="https://www.planet.com/contact-sales/" target="_blank">contact Planet</a>. 
 
+### EO Browser highlights
+
+3 area samples are available as EO Browser Highlight Visualisations.
 <br>
-<!---
-TODO: add link
--->
 
+<div class="container33">
+    <div class="image-card">
+    <a href='https://apps.sentinel-hub.com/eo-browser/?zoom=12&lat=44.7345&lng=-0.676&themeId=PLANET_SANDBOX&visualizationUrl=https%3A%2F%2Fservices.sentinel-hub.com%2Fogc%2Fwms%2Ff006c031-60da-4262-bdf7-6fc4f1532d13&datasetId=cc31cada-80d8-46fe-a746-43ac2f87b5da&fromTime=2017-01-01T00%3A00%3A00.000Z&toTime=2017-01-01T23%3A59%3A59.999Z&layerId=ABOVEGROUND-CARBON-DENSITY&demSource3D="MAPZEN"' target="_blank"><img src="FCM_LAOS.jpg" alt="EOB Highlight 1" class="imagette"></a>
+        <div class="info">
+            <div class="title">Khammouane, Laos</div>
+            <div class="text">
+                2023-09-21<br>
```

Comment:
I took visualization date, single day range, is that ok? FCD has much wider date range, not sure why?

Reply:
changed

Reply:
Nice!